pkg/cmd/extension/manager_test.go

```diff
@@ -720,26 +752,31 @@ func TestManager_UpgradeExtension_GitExtension_Pinned(t *testing.T) {
 }
 
 func TestManager_Install_local(t *testing.T) {
-	extManagerDir := t.TempDir()
+	dataDir := t.TempDir()
+	updateDir := t.TempDir()
 	ios, _, stdout, stderr := iostreams.Test()
-	m := newTestManager(extManagerDir, nil, nil, ios)
-	fakeExtensionName := "local-ext"
+	m := newTestManager(dataDir, updateDir, nil, nil, ios)
+	fakeExtensionName := "gh-local-ext"
```

Comment:
This creates a problem without being prefixed as extension manager works to ensure all extension directories are `gh-` prefixed for both the data and update directories.